hack/install_kustomize.sh

```diff
@@ -141,6 +141,10 @@ else
     releases=$(curl -s "$release_url" --header "Authorization: Bearer ${GITHUB_TOKEN}")
 fi
 
+if [[ $releases == *"Bad credentials"* ]]; then
+  echo "Authentication failed: Invalid GITHUB_TOKEN. Please check or remove your token."
+  exit 1
+  
```

Comment:
```suggestion
if [[ $releases == *"Bad credentials"* ]]; then
  echo "Authentication failed: Invalid GITHUB_TOKEN. Please check or remove your token."
  exit 1
fi  
```

I think you need to close the `if` condition

Reply:
Thanks @chansuke!